### PR TITLE
ctrl+ and ctr- for editor

### LINKED ISSUE #231: change the font size for doc editor
Ctr +- for ex. or add it to the menu

### PR DESCRIPTION
**Pull request by LianaHus**
fixes https://github.com/ethereum/mix/issues/231

### CODE REVIEW

File: qml/CodeEditor.qml [hunk 1]
```diff
@@ -95,6 +95,18 @@ Item {
...
 			}
+			if (event.modifiers().testFlag(Qt.Key_Control))//Qt::ControlModifier))
+				if (event.key() == Qt.Key_Plus)
+				{
+					codeEditor.font.pixelSize = codeEditor.font.pixelSize() * 0.20;
+				} else if (event.key() == Qt.Key_Minus)
+				{
+					codeEditor.font.pixelSize = codeEditor.font.pixelSize() / 0.20;
+				}
+
```
Comment (chriseth) [inline qml/CodeEditor.qml:106]: Unnecessary whitespace

**Comment by LianaHus** [comment]
@chriseth I need to cleanup some old code which is not used anymore. ctr+- works ok for us but Christoph can't use them. need to investigate.
